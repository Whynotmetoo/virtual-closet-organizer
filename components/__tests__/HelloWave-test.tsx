import * as React from 'react';
import { render } from '@testing-library/react-native';

import { HelloWave } from '../HelloWave';

it(`HelloWave component renders correctly`, () => {
  const tree = render(<HelloWave />).toJSON();

  expect(tree).toMatchSnapshot();
});
