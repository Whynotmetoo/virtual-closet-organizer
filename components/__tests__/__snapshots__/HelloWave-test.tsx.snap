// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWave component renders correctly 1`] = `
<View
  collapsable={false}
  jestAnimatedStyle={
    {
      "value": {
        "transform": [
          {
            "rotate": "0deg",
          },
        ],
      },
    }
  }
  style={
    {
      "transform": [
        {
          "rotate": "0deg",
        },
      ],
    }
  }
>
  <Text
    style={
      [
        {
          "color": "#11181C",
        },
        {
          "fontSize": 16,
          "lineHeight": 24,
        },
        undefined,
        undefined,
        undefined,
        undefined,
        {
          "fontSize": 28,
          "lineHeight": 32,
          "marginTop": -6,
        },
      ]
    }
  >
    ðŸ‘‹
  </Text>
</View>
`;
